src/components/results/sensitivityAnalysis/sensitivity-analysis-result-tab.js

```diff
@@ -0,0 +1,54 @@
+/**
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+import { useMemo, useState } from 'react';
+import PropTypes from 'prop-types';
+import Tabs from '@mui/material/Tabs';
+import Tab from '@mui/material/Tab';
+import SensitivityAnalysisTabs from './sensitivity-analysis-tabs';
+import PagedSensitivityResult from './paged-sensitivity-analysis-result';
+
+const SensitivityAnalysisResultTab = ({ studyUuid, nodeUuid }) => {
+    const [nOrNkIndex, setNOrNkIndex] = useState(0);
+    const [sensiKindIndex, setSensiKindIndex] = useState(0);
+
+    const pagedSensitivityResult = useMemo(
+        () => (
+            <PagedSensitivityResult
+                nOrNkIndex={nOrNkIndex}
+                sensiKindIndex={sensiKindIndex}
+                studyUuid={studyUuid}
+                nodeUuid={nodeUuid}
+            />
+        ),
+        [nOrNkIndex, sensiKindIndex, studyUuid, nodeUuid]
+    );
```

Comment:
is it really useful to memoize the component here ?

Reply:
no it is not necessary, I remove it. thank you